modules/manage/pages/cluster-maintenance/compaction-settings.adoc

```diff
@@ -26,6 +26,54 @@ Compaction policy may be applied to a cluster or to an individual topic. If both
 * `compact`: This triggers only cleanup of messages with multiple versions. A message that represents the only version for a given key is not deleted.
 * `compact,delete`: This combines both policies, deleting messages exceeding the retention period while compacting multiple versions of messages.
 
+== Tombstone record removal
+
+When a topic's messages should be deleted, for example as data is deleted from a source system, clients add a tombstone record to the message log. A tombstone contains the message key, and the value `null`. Tombstones signal to brokers and consumers that records with the same key prior to it in the log should be deleted. 
+
+You can specify how long Redpanda keeps these tombstones for compacted topics using both a cluster configuration property `tombstone_retention_ms` and a topic configuration property xref:reference:properties/topic-properties.adoc#deleteretentionms[`delete.retention.ms`]. If both are set, the topic-level tombstone retention policy overrides the cluster-level policy.
+
+[NOTE]
+====
+Redpanda does not remove tombstone records for compacted topics with Tiered Storage enabled. 
+
+You cannot enable `tombstone_retention_ms` if you have enabled any of the Tiered Storage cluster properties `cloud_storage_enabled`, `cloud_storage_enable_remote_read`, and `cloud_storage_enable_remote_write`.
+
+On the topic level, you cannot enable `delete.retention.ms` at the same time as the Tiered Storage topic configuration properties `redpanda.remote.read` and `redpanda.remote.write`.
+====
+
+To set the cluster-level tombstone retention policy, run the command:
+
+[,bash]
+----
+rpk cluster config set tombstone_retention_ms=100
+----
+
+You can unset the tombstone retention policy for a topic so it inherits the cluster-wide default policy:
+
+[,bash]
+----
+rpk topic alter-config <topic-name> --delete delete.retention.ms
+----
+
+To override the cluster-wide default for a specific topic:
+
+[,bash]
+----
+rpk topic alter-config <topic-name> --set delete.retention.ms=5
+----
+
+To disable tombstone removal for a specific topic:
+
+[,bash]
+----
+rpk topic alter-config <topic-name> --set delete.retention.ms=-1
+----
+
+Redpanda removes tombstones as follows:
+
+* For topics with a `compact,delete` cleanup policy: the standard garbage collection process, per the tombstone retention limit.
+* For topics with a `compact` only cleanup policy: tombstones must be seen at least twice by the compaction scheduler before they are removed. The `delete.retention.ms` or `tombstone_retention_ms` value sets the retention time for a tombstone, and in effect sets the time bound that a consumer has to read from the message log before they see an incomplete view of the log because tombstones were removed. 
```

Comment:
@WillemKauf two questions here - 

- Is there a possible configuration where the compaction schedule (`log_compaction_interval`?) conflicts or "interferes" with what's set for tombstone retention? For example, tombstone retention limit is set too low that the compaction scheduler is not going to see it at least twice?
- I'd love to have the two sentences in this paragraph be better tied together... Is it correct to say that: because the tombstone retention sets the time bound for consumers, tombstones can only be deleted after they are seen twice by the compaction scheduler?

Reply:
> For example, tombstone retention limit is set too low that the compaction scheduler is not going to see it at least twice?

No, because tombstones being marked as "seen" for the first time as well as the act of tombstone removal are both actions that occur during the compaction routine. There is no possible conflict here. 

> because the tombstone retention sets the time bound for consumers, tombstones can only be deleted after they are seen twice by the compaction scheduler?

I'm starting to think that "tombstones must be seen at least twice by the compaction scheduler before they are removed" is too much of an implementation detail to include here in the docs.

I think we could opt for the more succinct:

The retention time of tombstones in a compacted topic is controlled by the topic property `delete.retention.ms`, or the cluster property `tombstone_retention_ms`. The value used sets the time bound that a consumer has in order to see a complete view of the log with tombstones present before their removal.

Or something there-abouts.